### PR TITLE
Add OCaml versions

### PR DESCRIPTION
**Pull request by XVilka**

### CODE REVIEW

File: site/docs/install.md [hunk 1]
```diff
@@ -120,16 +128,17 @@ pacman -S ocaml
 Other Arch Linux packages related to OCaml are
 [listed here](https://www.archlinux.org/packages/?q=ocaml).
 
-## macOS
+## macOS
```
Comment (pmetzger) [inline site/docs/install.md]: Why add a space here? I think that's probably a mistake.

File: site/docs/install.md [hunk 2]
```diff
@@ -138,14 +147,16 @@ brew install opam
 apt install ocaml
 ```
```
Comment (pmetzger) [inline site/docs/install.md:141]: Is Fink still active? Maybe we should remove this mention entirely.

**Review by pmetzger — CHANGES_REQUESTED** [review]
Thanks for the changes! A few comments below...

**Comment by pmetzger** [comment]
Any idea why the CI system failed?

File: site/docs/install.md (hunk 2 above)
Comment (XVilka) [reply site/docs/install.md:141]: [Seems](http://www.finkproject.org/) it is. But is not yet added in repology: https://github.com/repology/repology/issues/243

**Comment by XVilka** [comment]
Fixed the space problem and added NetBSD mention. Also grouped FreeBSD, OpenBSD and NetBSD together. Travis CI error seems unrelated.

**Comment by Chris00** [comment]
Thanks.

**Pull request merged**